src/python/zquantum/core/interfaces/backend_test.py

```diff
@@ -49,28 +49,39 @@ def test_run_circuit_and_measure_correct_num_measurements(self):
     def test_get_expectation_values(self):
         # Given
         circuit = Circuit(Program(H(0), CNOT(0,1), CNOT(1,2)))
-        qubit_operator = IsingOperator('[]')
+        operator = IsingOperator('[]')
```

Comment:
This is an empty operator and then you have a `test_get_expectation_values_empty_op`, so it's slightly confusing for me.

Reply:
It's actually not an empty operator. It's an operator with only the identity term. I think this test is actually useful to make sure the identity case is handled properly

Reply:
👍